recipes/pti-gpu-unitrace/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pti-gpu-unitrace" %}
+# TODO: set version after original package is tagged
+{% set version = "0.0.1" %}
```

Comment:
You need a release before this can be merged.

Reply:
Talking to the maintainer to get a release tag

Reply:
Got the release!